### PR TITLE
Update riot.js version

### PR DESCRIPTION
**Pull request by marcioj**
Update the javascript file from 1.0.4 to 2.0.2

### CODE REVIEW

**Comment by adambutler** [comment]
@marcioj you'll also need to bump the version number please :)

**Comment by marcioj** [comment]
Hi @adambutler, sorry for the delay. Do you want that I change the gem version to `2.0.2` or `2.0.0`?

**Comment by adambutler** [comment]
It should always be the same version number as the javascript so `2.0.2`

**Comment by marcioj** [comment]
@adambutler done!

File: vendor/assets/javascripts/riot.js [hunk 1]
```diff
@@ -1,129 +1,688 @@
...
+// support AMD
+else if (typeof define === 'function' && define.amd)
+  define(function() { return riot })
+
+// support browser
+else
+  this.riot = riot
 
-})({});
+})();
```
Comment (adambutler) [inline vendor/assets/javascripts/riot.js]: @marcioj Can you add a line break here please ([reason](http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline)). [Heres](https://github.com/editorconfig/editorconfig/wiki/Newline-at-End-of-File-Support) how to get your editor to always do this :)

Comment (marcioj) [inline vendor/assets/javascripts/riot.js]: Thanks for the tip, I updated :)

**Comment by adambutler** [comment]
Will push to rubygems now :+1: Thank you!

**Pull request merged**